packages/lit-html/src/lit-html.ts

```diff
@@ -744,7 +759,7 @@ export class NodePart {
       // set its value, rather than replacing it.
       (node as Text).data = value as string;
     } else {
-      this._commitNode(new Text(value as string));
+      this._commitNode(document.createTextNode(value as string));
```

Comment:
Is it worth including this as a dom shim?

Reply:
I don't think so... it's such a small change in only one place.